### ISSUE TITLE
tilegenmanager.prod.bgdi.ch doesn't work (is this project still active?)

### ISSUE THREAD

**Issue by fnicollet** [geoadmin/3d-forge#169]
Hello,

We are looking at a solution for creating custom terrain meshes (from a TIFF source or similair) usable with Cesium. Basically, we need a terrain mesh generator, which seem to be the aim of this project.
We found a link to this project in a Cesium forum. 

In the readme, there is a link to:
https://tilegenmanager.prod.bgdi.ch/index/index
Which returns an error:
Error 404 Unknown virtual host

But looking at the issues and activity of this project, it looks like it is not active anymore?

Thanks,
Fabien

**Comment by gjn** [comment]
Hi Fabien

The link below probably was never active.

There's a github project [1] we published that we used to create
the terrain out of an already created mesh. It uses shapefile/postgis 
backend
to create the terrain pyramid. This project is as is and we don't offer 
real
support for it.

To created meshes: this is actually the hard part as most mesh creating 
programs
are using local coordinate systems - and not WGS84, as needed in cesium. 
We used
proprieraty software (I belive ESRI) to achieve the meshing and then 
transformed to
WGS84.

Hope this helps.

Gilbert

[1]https://github.com/geoadmin/3d-forge
